README.md

```diff
@@ -1,20 +1,12 @@
 # somatic-cnvs
-Workflows for somatic copy number variant analysis
 
-## Running the Somatic CNV WDL
+### Purpose :
+Workflows for somatic copy number variant analysis.
 
-### Which WDL should you use?
+### cnv_somatic_panel_workflow :
+Builds a panel of normals (PoN) for the cnv pair workflow.
 
-- Building a panel of normals (PoN): ``cnv_somatic_panel_workflow.wdl``
-- Running a matched pair: ``cnv_somatic_pair_workflow.wdl``
-
-#### Setting up parameter json file for a run
-
-To get started, create the json template (using ``java -jar wdltool.jar inputs <workflow>``) for the workflow you wish to run and adjust parameters accordingly.
-
-*Please note that there are optional workflow-level and task-level parameters that do not appear in the template file.  These are set to reasonable values by default, but can also be adjusted if desired.*
-
-#### Required parameters in the somatic panel workflow
+#### Requirements/Expectations
 
 Important: The normal_bams samples in the json can be used test the wdl, they are NOT to be used to create a panel of normals for sequence analysis. For instructions on creating a proper PON please refer to user the documents https://software.broadinstitute.org/gatk/documentation/ .
```

Comment:
Is there a specific document we can point to? Quick googling turns up a bunch of options, maybe this one is best:
https://gatkforums.broadinstitute.org/gatk/discussion/11053/panel-of-normals-pon

Reply:
I can add this one as well, https://gatkforums.broadinstitute.org/dsde/discussion/11682#2